fastapi_jsonrpc/__init__.py

```diff
@@ -6,7 +6,7 @@
 from collections import ChainMap
 from contextlib import AsyncExitStack, AbstractAsyncContextManager, asynccontextmanager, contextmanager
 from json import JSONDecodeError
-from types import FunctionType, CoroutineType
+from types import FunctionType, Coroutine
```

Comment:
I think you mean `typing.Coroutine` instead `types.Coroutine`, but we should use collections.abc.Coroutine, cause typing.Coroutine deprecated since Python 3.9

https://devdocs.io/python~3.10/library/typing#typing.Coroutine

Please, check it again

```bash
❯ python3
Python 3.10.2 (main, Jan 15 2022, 19:56:27) [GCC 11.1.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from types import Coroutine
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'Coroutine' from 'types' (/usr/lib/python3.10/types.py)
```

Reply:
hey!
yep, sorry, I didn't check it properly
fixed